femwell/eme.py

```diff
@@ -281,8 +281,7 @@ def compute_total_S_matrix(meshnames, mesh_info_dict, lengths, wavelength, num_m
         background_padding=(2.0, 2.0, 2.0, 2.0), # how much of backgorund tag to add to each side of the simulatoin
         filename=f"mesh_x_{x1}.msh",
     ))
-    meshnames.append(f"mesh_x_{x1}.msh")
-
+    meshnames = [f"mesh_x_{x1}.msh"]
```

Comment:
I'd guess this is already prepared for being put in a loop? @simbilod Therefore, I'd suggest not changing this?
```suggestion
    meshnames.append(f"mesh_x_{x1}.msh")
```